frontend/pages/project.tsx

```diff
@@ -1,26 +1,82 @@
+import Link from 'next/link';
+import { fill_project, fill_user } from '../model';
+import Image from 'next/image';
+
+const Project: React.FC<{ projects: fill_project[], users: fill_user[] }> = ({ projects, users }) => {
+    const getStatus = (status: string): string => {
+        switch (status) {
+        case 'Pending':
+            return 'bg-gray-500'; 
+        case 'in Progress':
+            return 'bg-yellow-500'; 
+        case 'Done':
+            return 'bg-green-800'; 
+        case 'Archived':
+            return 'bg-green-300'; 
+        case 'Refused':
+            return 'bg-red-500'; 
+        case 'req finish':
+            return 'bg-purple-500'; 
+        default:
+            return 'bg-slate-100';
+        }
+    };
+    const getUserInfo = (leaderId: number): fill_user | undefined => {
+        return users.find(user => user.id === leaderId);
+    };
 
-const Project: React.FC<{ projects: string[] }> = ({ projects }) => {
     return (
-        <>
-            <h1>Project</h1>
-            <ul>
-                {projects.map((project, index) => (
-                    <li key={index}>{project}</li>
-                ))}
-            </ul>
-        </>
+        <div className="h-auto p-2 m-4 max-w-screen break-words">
+            <div className="p-2 flex items-center justify-between flex-wrap">
+                <h1 className="text-5xl text-slate-900">Projects</h1>
+                <Link href="/create-project" title="Create project" className="text-xl text-slate-900 hover:text-2xl focus:font-semibold font-bold justify-content: space-between">CREATE PROJECT</Link>
+            </div>
+            <div>
+                <ul data-te-infinite-scroll-init className="overflow-y-scroll gap-6 max-w-screen md:grid md:grid-cols-2 lg:grid-cols-3 xl:grid-cols-4">
+                    {projects?.map((project: fill_project, idx: number) => {
+                        const leaderUser = getUserInfo(project.leader_id);
+                        return (
+                            <li key={idx} className="m-4 flex flex-col justify-between relative rounded-xl bg-white text-black shadow-xl transition ease-in-out delay-100 hover:scale-105 duration-250">
+                                {/* <Link href={`/project/${project.id}`} title={project.name}> */}
+                                <Image src={project.image} alt={project.name} width={300} height={300} className="object-cover rounded-t-lg aspect-square mx-auto w-40 md:w-full"></Image>
+                                <div className="m-2 break-words">
+                                    <h1 className="text-center text-4xl break-words">{project.name}</h1>
+                                    <p className="line-clamp-5 mt-2 text-lg break-words">{project.description}</p>
+                                    <p className="italic font-light mt-8 p-2 text-sm break-words">
+                                        {leaderUser ? (leaderUser.email) : ('N/A')}
```

Comment:
N/A stands for not applicable and should not be used here
do not use `break-words`, email should be displayed in a single line